src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, AttributesDrawing {
```

Comment:
Don't use abstract classes to set behavior for classes. In this context, 'Figure' should not be abstract if it's only used to share the 'color' field. Consider using an interface if you want to enforce certain methods, or a concrete class if you want to share common fields or behavior.